backend/src/main/resources/application-prod.yml

```
@@ -25,6 +25,16 @@ spring:
     user: ${DB_USERNAME}
     password: ${DB_PASSWORD}
 
+spring:
```

YAMLlint (1.38.0)

[error] duplication of key "spring" in mapping

(key-duplicates)